invenio_rdm_pure/source/reports.py

```diff
@@ -1,63 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 Technische Universität Graz
+#
+# invenio-rdm-pure is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""File description."""
+
 from datetime import date, datetime
 
 from source.general_functions import add_spaces, check_if_directory_exists, current_time
 
 from setup import dirpath, log_files_name
 
-report_templates = {
```

Comment:
why removing report_templates. there is a reference to it in line 30

Reply:
I will check it.